src/shared/components/case-viewer/case-viewer.component.ts

```diff
@@ -39,7 +39,9 @@ export class CaseViewerComponent implements OnInit, OnDestroy {
   BANNER = DisplayMode.BANNER;
 
   caseDetails: CaseView;
+  @Type(() => CaseTab)
```

Comment:
is this a leftover? Usually this annotation should be used only on domain classes, not on a component